blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -219,4 +127,74 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws
             return pipeline;
         }
     }
+    static class Processor {
```

Comment:
I guess should have asked, how would you have implemented it?  Is there a better way to do it?